library/src/main/java/com/proxerme/library/connection/info/request/EntryCoreRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.proxerme.library.connection.info.request;
+
+import android.support.annotation.IntRange;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.afollestad.bridge.Form;
+import com.afollestad.bridge.Response;
+import com.proxerme.library.connection.ProxerRequest;
+import com.proxerme.library.connection.info.result.EntryCoreResult;
+import com.proxerme.library.info.ProxerTag;
+import com.proxerme.library.info.ProxerUrlHolder;
+
+/**
+ * Request object for the core data of an entry by an id (Anime or Manga).
+ *
+ * @author Desnoo
+ */
+public class EntryCoreRequest extends ProxerRequest<EntryCoreResult> {
+
+    private static final String ENTRY_CORE_URL = "/api/v1/info/entry";
+
+    private static final String ENTRY_CORE_ID = "id";
+
+    private int id;
```

Comment:
Integer is not a good datatype for IDs on Proxer as those can have leading zeros. (01234 for example, which would be parsed to 1234 for an Integer) String would be better.